examples/colab/wav2vec2_saved_model_finetuning.ipynb

```diff
@@ -0,0 +1,2441 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    pbar = tqdm(range(EPOCHS), total=EPOCHS)
</pre><p><code>tqdm</code> comes from your wav2vec repo, right? It might be good to give it a more clear name if it's going to be relied on externally like this.</p><p> </p><p>Similarly consider renaming <code>pbar</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/hub/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
tqdm is not coming from my project, rather using [this package](https://tqdm.github.io).